src/reducer.js

```diff
@@ -149,9 +150,9 @@ export const timestampsReducer = (state = {}, { type, path }) => {
 const createDataReducer = (actionKey = 'data') => (state = {}, action) => {
   switch (action.type) {
     case SET:
-      return set(state, getDotStrPath(action.path), action[actionKey])
+      return set(getDotStrPath(action.path), action[actionKey], state)
```

Comment:
You are totally right! I didn't notice the `fp` in the import.

[This lodash issue confirms this usage](https://github.com/lodash/lodash/issues/1696#issuecomment-269394600). Thanks for the contribution!